src/components/KMultiselect/KMultiselect.vue

```diff
@@ -211,7 +215,7 @@
           v-for="item, idx in visibleSelectedItemsStaging"
           :key="`${item.key ? item.key : idx}-badge`"
           class="mr-1 mt-2"
-          dismissable
+          :dismissable="item.selected && !item.disabled"
```

Comment:
question: just making sure this logic looks correct?

You can remove the item if it IS selected, and NOT disabled. If this sounds good, just resolve this comment 😬

Reply:
That sounds right.